### ISSUE TITLE
Missing docs for informative-docs > excludedTags option

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [gajus/eslint-plugin-jsdoc#1214]
## Motivation

`jsdoc/informative-docs` will scan the body and all tags of a JSDoc tag for informative-ness. This is good and intended in most cases. But sometimes, there are tags that are used as categorization / metadata and only coincidentally match the name of the construct they're documenting. For example:

```js
/**
 * Calculates and retrieves the compiler options for the current project.
 * @category Compiler Options
 */
function getCompilerOptions() {
  // ...
}
```

Standalone reproduction of that snippet for reference: https://github.com/JoshuaKGoldberg/repros/tree/jsdoc-informative-docs-category-tag

## Current behavior

```plaintext
  3:1  error  This tag description only repeats the name it describes  jsdoc/informative-docs
```

## Desired behavior

Can we add an option to the rule to allow a list of tags to be ignored? Say, `ignoredTags`?

```js
"jsdoc/informative-docs": ["error", {
  "ignoredTags": ["category"]
}]
```

## Alternatives considered

An inline disable comment is doable, but clutters the code and is generally inconvenient.

**Comment by brettz9** [comment]
There is already an `excludedTags` array option... It just isn't currently documented.

**Comment by JoshuaKGoldberg** [comment]
Aha! #1153 - that must be why it didn't come up in my searches. Nice. Thanks - will send a docs PR.
